src/main/steps/common/task-list/components/notification-banner/utils.ts

```diff
@@ -97,6 +97,20 @@ export const showNotification = (notificationType: NotificationType, caseData: C
 
   return notificationId ? findNotification(caseData, notificationId)?.show ?? false : false;
 };
+export function showPreDashBoardNotification(notificationType: NotificationType, caseData: CaseWithId): boolean {
+  switch (notificationType) {
```

Comment:
Initialise show as false and Instead of switch use if else and return show at the end.